ts/create-kpt-functions/src/cmd/type_create.ts

```diff
@@ -87,6 +67,73 @@ export async function typeCreate(packageDir: string) {
   log(format.finishMarker(desc));
 }
 
+// chooseTypeSource lets the user choose the source for the kubernetes types and returns
+// numeric choice
+function chooseTypeSource(): number {
+  const defaultContext = 0;
+  const ch = validator.getValidString(
+    () =>
+      question(
+        `> Kubernetes Types used to generate Typescript library (0) \n` +
```

Comment:
More wordsmithing:

`What's the source of OpenAPI document defining the Kubernetes types? This will be used to generate Typescript library.

[0] Use built-in document (No cluster required)
[1] Download the document from a Kubernetes clusters`

Reply:
@frankfarzan Done